tdrs-backend/tdpservice/data_files/models.py

```diff
@@ -164,8 +164,6 @@ def prog_type(self):
         # e.g., 'SSP Closed Case Data'
         if self.section.startswith('SSP'):
             return 'SSP'
-        elif self.section.startswith('Tribal'):
```

Comment:
@elipe17 just curious why we're dropping this. i didnt observe any weird behavior as a result but wondering if we removed this as part of dropping the program type filter?

Reply:
I removed it because it was an unnecessary branch since tribal and tanf both result in TANF